codersdk/features.go

```diff
@@ -0,0 +1,29 @@
+package codersdk
+
+type Entitlement string
+
+const (
+	EntitlementEntitled    Entitlement = "entitled"
+	EntitlementGracePeriod Entitlement = "grace_period"
+	EntitlementNotEntitled Entitlement = "not_entitled"
+)
+
+const (
+	FeatureUserLimit = "user_limit"
+	FeatureAuditLog  = "audit_log"
+)
+
+var AllFeatures = []string{FeatureUserLimit, FeatureAuditLog}
```

Comment:
nit: `Features` would implicitly assume all `Feature` type values are included in the array.

Reply:
`AllFeatureNames`?

Reply:
I phrased this poorly. I'm proposing `Features` instead of `AllFeatures`.

Reply:
Ah, fine with me.

Reply:
But note, these are of type `string` not type `Feature`

Reply:
I went with `FeatureNames` since it's a slice of strings, not a slice of `Feature`